client/checkout/express-checkout-buttons.scss

```
@@ -33,6 +33,19 @@
 			margin-top: 4px;
 		}
 	}
+
+	// Add specific spacing for product pages in both traditional and block-based layouts
+	.woocommerce .type-product &,
+	.wp-block-add-to-cart-form & {
+		margin-top: 1em;
+		margin-bottom: 1em;
+	}
+
+	// Add spacing for shortcode cart
+	.wc-proceed-to-checkout & {
```

Could these two block be simplified by doing this?
```suggestion
   	.wc-proceed-to-checkout &,
	.woocommerce .type-product &,
	.wp-block-add-to-cart-form & {
		margin-top: 1em;
		margin-bottom: 1em;
	}
```

---

Yes makes sense to simplify it